apps/pie-storybook/stories/pie-link.stories.ts

```diff
@@ -88,16 +93,19 @@ const linkStoryMeta: LinkStoryMeta = {
 export default linkStoryMeta;
 
 const Template = ({
-    href, target, rel, size, variant, isBold, isStandalone, slot,
+    href, target, rel, size, variant, isBold, isStandalone,
```

Comment:
Would you mind listing each of these on separate lines please?

Reply:
yess